docs/src/test-advanced.md

```diff
@@ -201,22 +201,24 @@ export const test = base.extend<{ saveLogs: void }>({
 });
 ```
 
-## Launching a development web server during the tests
```

Comment:
You can leave the old title and refer to it as a web server. They will figure out it can be used for other purposes.